nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -347,7 +347,7 @@ sub in {
 
     # Skip special filesystems.
     next if in($mountPoint, "/proc") || in($mountPoint, "/dev") || in($mountPoint, "/sys") || in($mountPoint, "/run") || $mountPoint eq "/var/lib/nfs/rpc_pipefs";
-    next if $mountPoint eq "/var/setuid-wrappers";
+    next if $mountPoint eq "/run/wrappers";
```

Comment:
Yeah it isn't, this was legacy from when I simply did a find-replace. Thanks for catching this.